tests/e2e/pageobjects/ide/OpenDialogWidget.ts

```diff
@@ -20,13 +20,13 @@ import { TimeoutConstants } from '../../TimeoutConstants';
 export enum Locations {
     Theia = 'theia',
     Home = 'home',
-    Root = '/'
+    Root = '/',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.